app/__init__.py

```diff
@@ -1,7 +1,49 @@
 from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+from dotenv import load_dotenv
+import os
+
+db = SQLAlchemy()
+migrate = Migrate()
+load_dotenv() #need this to point to our .env file which connects to the dbs
 
 
 def create_app(test_config=None):
     app = Flask(__name__)
 
+    if not test_config:
+        app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+        app.config['SQLALCHEMY_DATABASE_URI'] = os.environ.get(
+            "SQLALCHEMY_DATABASE_URI")
+    else:
+        app.config["TESTING"] = True
+        app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+        app.config["SQLALCHEMY_DATABASE_URI"] = os.environ.get(
+            "SQLALCHEMY_TEST_DATABASE_URI")
+
+
+
+    db.init_app(app)
+    migrate.init_app(app, db)
+
+    from .routes import planets_bp
+    app.register_blueprint(planets_bp)
+
+    from app.models.planet import Planet
```

Comment:
The blueprints already import the models so we don't actually need the model import in this file anymore.